docs/source/admin/traffic_portal/usingtrafficportal.rst

```diff
@@ -367,15 +367,18 @@ Topology
 
 	'Topology' Menu
 
+.. _tp-configure-cache-groups:
+
 Cache Groups
 ------------
 'Cache Groups' are sets of cache servers, typically grouped by geographic proximity. This menu allows user to add or remove caches from Cache Groups as well as creating, updating and deleting Cache Groups themselves. Each entry in the table of Cache Groups on this page has the following fields:
 
-:Name:       The full name of this Cache Group
-:Short Name: A shorter, more human-friendly name for this Cache Group
-:Type:       The Type of this Cache Group (see :ref:`tp-configure-types`)
-:Latitude:   A geographic latitude assigned to this Cache Group
-:Longitude:  A geographic longitude assigned to this Cache Group
+:Name:                  The full name of this Cache Group
```

Comment:
"Cache Group" should link to the glossary

Reply:
done